backend/src/user/user.controller.ts

```diff
@@ -7,43 +7,70 @@ import {
   ParseIntPipe,
   Patch,
   Post,
+  UseGuards,
 } from '@nestjs/common'
-import { ApiBody, ApiTags } from '@nestjs/swagger'
+import { ApiBearerAuth, ApiBody, ApiOperation, ApiTags } from '@nestjs/swagger'
+import { Role } from '@prisma/client'
+import { HasRoles } from '../auth/roles.decorator'
+import { RolesGuard } from '../auth/roles.guard'
+import { JwtAuthGuard } from '../auth/jwt-auth.guard'
+import { RequestUser } from '../auth/user.decorator'
 import { CreateUserDto, UpdateUserDto } from './dto/user.dto'
 import { UserService } from './user.service'
+import { Public } from '../auth/public.decorator'
 
+@Public()
```

Comment:
To trzeba wywalić, bo przez to wszystkie endpointy są publiczne. @Public ma byc tylko przy loginie na ten moment